pages/common/docker-build.md

```diff
@@ -0,0 +1,28 @@
+# docker build
+
+> Build an image from a Dockerfile.
+> More information: <https://docs.docker.com/engine/reference/commandline/build/>.
+
+- Build an docker image using the Dockerfile in current directory:
+
+`docker build .`
+
+- Build an docker image with URL:
+
+`docker build {{github.com/creack/docker-firefox}}`
+
+- Build an docker image and tag it:
+
+`docker build --tag {{name:tag}} .`
+
+- Do not use cache when building the image:
+
+`docker build --no-cache --tag {{name:tag}} .`
+
+- Build an docker image using a specific Dockerfile:
```

Comment:
Perhaps this will read better?
```suggestion
- Build a docker image using a specific Dockerfile:
```

Also, is there a long form of the `-f` argument?